README.md

```diff
@@ -1,262 +1,44 @@
-* `for i in range(x, y): ...` : x and y must be nonzero positive constant integers, ie. specified at compile time
-* `for i in range(start, start + x): ...` : start can be any expression, though it must be the exact same expression in both places. x must be a nonzero positive constant integer.
-
-In all three cases, it's possible to statically determine the maximum runtime of a loop. Jumping out of a loop before it ends can be done with either `break` or `return`.
-
-Regarding byte array literals, unicode strings like "这个傻老外不懂中文" or "Я очень умный" are illegal, though those that manage to use values that are in the 0...255 range according to UTF-8, like "¡très bien!", are fine.
-
-Code examples can be found in the `test_parser.py` file.
-
-### Visibility
-
-* `public(type)`: publicly visible state variable
-
-### Planned future features
-
-* Declaring external contract ABIs, and calling to external contracts
-* A mini-language for handling num256 and signed256 values and directly / unsafely using opcodes; will be useful for high-performance code segments
-* Smart optimizations, including compile-time computation of arithmetic and clamps, intelligently computing realistic variable ranges, etc
-
-### Code example
-
-```python
-funders: {sender: address, value: wei_value}[num]
-nextFunderIndex: num
-beneficiary: address
-deadline: timestamp
-goal: wei_value
-refundIndex: num
-timelimit: timedelta
-    
-# Setup global variables
-def __init__(_beneficiary: address, _goal: wei_value, _timelimit: timedelta):
-    self.beneficiary = _beneficiary
-    self.deadline = block.timestamp + _timelimit
-    self.timelimit = _timelimit
-    self.goal = _goal
-    
-# Participate in this crowdfunding campaign
-@payable
-def participate():
-    assert block.timestamp < self.deadline
-    nfi = self.nextFunderIndex
-    self.funders[nfi] = {sender: msg.sender, value: msg.value}
-    self.nextFunderIndex = nfi + 1
-    
-# Enough money was raised! Send funds to the beneficiary
-def finalize():
-    assert block.timestamp >= self.deadline and self.balance >= self.goal
-    selfdestruct(self.beneficiary)
-    
-# Not enough money was raised! Refund everyone (max 30 people at a time
-# to avoid gas limit issues)
-def refund():
-    assert block.timestamp >= self.deadline and self.balance < self.goal
-    ind = self.refundIndex
-    for i in range(ind, ind + 30):
-        if i >= self.nextFunderIndex:
-            self.refundIndex = self.nextFunderIndex
-            return
-        send(self.funders[i].sender, self.funders[i].value)
-        self.funders[i] = None
-    self.refundIndex = ind + 30
-```
+# About
+Viper is an experimental programming language that aims to be a simple,
+Python-based solution to building easy-to-read contracts on the Ethereum network.
+It is designed to avoid common issues when writing contracts,
+and make logical, clear contracts as easily as possible.
+For more information, see [Design Principles](no-link) 
```

Comment:
It is designed to avoid common issues when writing contracts, `by making writing logical,`

Reply:
I will change to:
It is designed to avoid common issues ` and mistakes with smart-`contracts` by design`, `and make writing` logical, clear `smart-`contracts as `easy` as possible.

What do you think?